packages/clerk-js/src/ui/ImpersonationFab/ImpersonationFab.tsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+
+import { mqu } from '../../ui/styledSystem';
+import { getFullName, getIdentifier } from '../../ui/utils';
+import { useCoreClerk, useCoreSession } from '../contexts';
+import {
+  Col,
+  descriptors,
+  Flex,
+  Icon,
+  Link,
+  LocalizationKey,
+  localizationKeys,
+  Text,
+  useAppearance,
+} from '../customizables';
+import { Portal } from '../elements/Portal';
+import { Eye } from '../icons';
+
+type EyeCircleProps = {
+  width: string;
+  height: string;
+};
+
+const EyeCircle = ({ width, height }: EyeCircleProps) => {
+  return (
+    <Col
+      id='cl-impersonationEye'
+      elementDescriptor={descriptors.impersonationFabIconContainer}
+      justify='center'
+      sx={t => ({
+        width,
+        height,
+        transition: `transform ${t.transitionDuration.$slowest} ease`,
+        backgroundColor: t.colors.$danger500,
+        borderRadius: t.radii.$circle,
+      })}
+    >
+      <Icon
+        elementDescriptor={descriptors.impersonationFabIcon}
+        sx={{
+          margin: 'auto',
+        }}
+        icon={Eye}
+        size={undefined}
```

Comment:
❓ Can you please elaborate?

Reply:
Did not want a default width/height because the svg was getting cropped. Its width is not equal to its height by design.